tests/data_context/test_data_context_datasources.py

```diff
@@ -453,3 +453,21 @@ def test_list_datasources() -> None:
     context.sources.add_pandas(datasource_name)
 
     assert len(context.list_datasources()) == 2
+
+
+@pytest.mark.unit
+def test_get_available_data_assets_names(test_df_pandas, empty_data_context) -> None:
```

Comment:
As `empty_data_context` scaffolds the filesystem as part of setup, I wouldn't consider this a unit test but rather an integration test - getting true, pure unit tests with the context is difficult work